From 7fd33a6d24da6a82b830552999b2332140f556d4 Mon Sep 17 00:00:00 2001
From: Jim Meyering <meyering@redhat.com>
Date: Mon, 26 Mar 2012 21:52:08 +0200
Subject: [PATCH 04/89] tests: remove bogus envvar setting from
 t0300-dos-on-gpt.sh

* tests/t0300-dos-on-gpt.sh: Remove envvar setting that effectively
disabled testing(only in this test) of simulated sector sizes smaller
than 4KiB.
Also, use a much smaller backing file.
---
 tests/t0300-dos-on-gpt.sh | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/tests/t0300-dos-on-gpt.sh b/tests/t0300-dos-on-gpt.sh
index 41bc391..fbe8d7c 100755
--- a/tests/t0300-dos-on-gpt.sh
+++ b/tests/t0300-dos-on-gpt.sh
@@ -17,13 +17,11 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 . "${srcdir=.}/init.sh"; path_prepend_ ../parted
-
-PARTED_SECTOR_SIZE=4096
-export PARTED_SECTOR_SIZE
+ss=$sector_size_
 
 dev=loop-file
 # create a backing file large enough for a GPT partition table
-dd if=/dev/null of=$dev seek=4001 2> /dev/null || framework_failure
+dd if=/dev/null of=$dev bs=$ss seek=80 2> /dev/null || framework_failure
 
 # create a GPT partition table
 parted -s $dev mklabel gpt > out 2>&1 || fail=1
-- 
1.8.5.3

